test/ShellTest.php

```diff
@@ -516,16 +519,27 @@ public function testWriteExceptionVerboseButNotReallyBecauseItIsABreakException(
         $shell = new Shell($this->getConfig());
         $shell->setOutput($output);
 
-        $shell->writeException(new BreakException('yeah.'));
+        $shell->writeException(new BreakException('yeah'));
         \rewind($stream);
-        $this->assertSame('Exit:  yeah.'.\PHP_EOL, \stream_get_contents($stream));
+        $this->assertSame(<<<EOF
+
+   INFO  yeah.
+
+
+EOF, \stream_get_contents($stream));
```

Comment:
in older PHP, you had to end the heredoc with a `;`. So this would need to be something like

```suggestion
        $expected = <<<EOF

   INFO  yeah.


EOF;
        $this->assertSame($expected, \stream_get_contents($stream));
```